tutorials/7_Advanced_ChangeCode.md

```diff
@@ -10,16 +10,36 @@ Florian Humpenöder (<humpenoeder@pik-potsdam.de>), Lavinia Baumstark (<baumstar
 
 Technical Structure
 =====================
-The REMIND-code is structured in a modular way. The technical structure looks as follows: At the top level you find the folders config, core, modules and scripts. The overall structure is built in the file main.gms. All settings and configuration information are given in the config folder.The core folder contains all files that are part of the core of the REMIND model. For each module there exists a sub-folder in the modules folder. Helpful scripts for e.g. starting a run or analysing results you find in the scripts folder.
+The REMIND-code is structured in a modular way. The technical structure looks as follows: at the top level you find the folders `config`, `core`, `modules`, and `scripts`. The overall structure is built in the file `main.gms`. All settings and configuration information are given in the `config` folder. The `core` folder contains all files that are part of the core of the REMIND model. For each module there exists a sub-folder in the `modules` folder. Helpful scripts for e.g. starting a run or analysing results you find in the `scripts` folder.
 
-In the main.gms the technical structure of REMIND can be found. First, the *.gms files from the core folder are included and afterward the *.gms files from the activated module realization, beginning with the one with the smallest module-number. The technical structure of REMIND looks as follows:
+In the `main.gms` file the technical structure of REMIND can be found. First, the `*.gms` files from the core folder are included and afterward the `*.gms` files from the activated module realization, beginning with the one with the smallest module-number. The technical structure of REMIND looks as follows:
 
-<img src="figures/technical_structure.png" alt="Technical Structure of REMIND" width="100%" />
```

Comment:
right, much better to add it here directly, please delete the figure which is not used any more

Reply:
The image was also used in `main.gms`, I replaced it there as well and removed it from the repo.